packages/lib/src/Login.wc.svelte

```diff
@@ -45,11 +45,13 @@
 </script>
 
 <Base let:api let:t>
-  {#await checkLoggedIn(api) then _}
+  {#await checkLoggedIn(api)}
+    <Avatar loading={true} size="sm"></Avatar>
```

Comment:
I was thinking that the checkLoggedIn would be so fast you wouldn't notice, but if your access-token is expired it's probably not true. Anyhow this is a better solution.

Reply:
If you log in and reload the page it's never that fast, you always get the visible flash paint.

If I find time I'll refactor this even more. We want to avoid flashing content at any cost since this will end up on various websites around the world.